app.py

```diff
@@ -1018,7 +1018,7 @@ def showPageOptions():
         tmp = sp.call('clear', shell=True)
         refreshDatabase()
         print("Choose an option from below: ")
```

Comment:
If I understand correctly,  `refreshDatabase()` will be unnecessary for our CLI if we check for integrity while updating and deleting. It's there in insertions; if it's there in delete and update as well, we probably don't need this.

Reply:
We can think about it and do change accordingly in another PR.